supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -83,18 +76,18 @@ For documentation questions, contact rhos-docs@redhat.com.
 [discrete]
 [[red-hat-openstack-platform]]
 ==== image:images/yes.png[yes] Red Hat OpenStack Platform (noun)
-*Description*: Spell out in full. This product name applies to Red Hat OpenStack Platform 8 and later versions.
+*Description*: On the first use in a module, use the complete product name and the abbreviation in parentheses: _Red Hat OpenStack Platform (RHOSP)_. After the first instance, use: _RHOSP_. This product name applies to RHOSP 8 and later versions.
 
 *Use it*: yes
 
-*Incorrect forms*: RHOS, RHOSP, RH-OSP
+*Incorrect forms*: OpenStack Platform, RHOS, RH-OSP
 
 *See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]
 
 [discrete]
 [[undercloud]]
 ==== image:images/yes.png[yes] undercloud (noun)
-*Description*: The undercloud is the main director node. It is a single-system OpenStack installation that includes components for provisioning and managing the OpenStack nodes that form your OpenStack environment (the overcloud). Write in lower case.
+*Description*: The undercloud is the director node. It is a single-system RHOSP installation that includes components for provisioning and managing the RHOSP nodes that form your environment (the overcloud). Write in lowercase.
```

Comment:
```suggestion
*Description*: The undercloud is the director node. It is a single-system within the RHOSP installation that includes components for provisioning and managing the RHOSP nodes that form your environment, known as the overcloud. Write in lowercase.
```

@bergerhoffer Do we include cross-references in glossaries? If we do, we can link `overcloud`

Reply:
Hi @ndeevy . Would you be OK to incorporate the above suggestion?

@porueesq , you OK to approve this PR after this thread is marked as "Resolved"?

Reply:
Sure @dfitzmau . I'll keep my eyes open :eyes:

Reply:
Hi @dfitzmau that addition was made in the **Files changed** tab, I just hadn't hit **Resolve** here so doing that now. Thanks!

Reply:
Thanks. I had to expand the Files Changes preview to see the link. 

Good work! I'll merge this PR.